docs/inference.rst

```diff
@@ -108,7 +108,8 @@ An example configuration file (named ``inference.ini``) is::
     [static_args]
     ; waveform parameters that will not change in MCMC
     approximant = IMRPhenomPv2
-    f_lower = 19.0
+    f_lower = 18
```

Comment:
I'm wary of starting the waveform at the same frequency as the lower cutoff of the matched filter, in case there is any initial weirdness at the start of the waveform. For example, with time domain waveforms, you need to taper the start of the waveform. Better to do that out of band, so it doesn't affect your results. I know the templates are frequency domain, but I prefer to err on the side of caution. I didn't realize they had used 20Hz in the O1 PE. In any event, setting `f_ref = 20` should accomplish the same thing. I've adjusted f_ref to that value.